opendut-edgar/src/service/start.rs

```diff
@@ -171,7 +174,68 @@ async fn handle_stream_message(
                 }
                 _ => ignore(message),
             }
-            Message::ApplyVpnConfig(_) => {}
+            Message::ApplyPeerConfiguration(message) => match message {
+                ApplyPeerConfiguration { configuration: Some(configuration) } => {
+                    log::info!("Received configuration: {configuration:?}");
+                    match PeerConfiguration::try_from(configuration) {
+                        Err(error) => log::error!("Illegal PeerConfiguration: {error}"),
+                        Ok(configuration) => {
+                            for executor in configuration.executors.executors {
+                                match executor {
+                                    ExecutorDescriptor::Executable => log::warn!("Executing Executable not yet implemented."),
+                                    ExecutorDescriptor::Container {
+                                        engine,
+                                        name,
+                                        image,
+                                        volumes,
+                                        devices,
+                                        envs,
+                                        ports,
+                                        command,
+                                        args
+                                    } => {
+                                        let engine = match engine {
+                                            Engine::Docker => { "docker" }
+                                            Engine::Podman => { "podman" }
+                                        };
+                                        let mut cmd = Command::new(engine);
+                                        if let Some(command) = command {
+                                            cmd.arg(command.value());
+                                        } else {
```

Comment:
You always have to run `podman run` the command in our case is the binary inside the container which should be executed!

Check out the docs [here](https://docs.podman.io/en/latest/markdown/podman-run.1.html). The synopsis looks like: `podman run [options] image [command [arg …]]`

Example: `podman run -i -t fedora /bin/bash`

The example above will create an interactive container from the `fedora` image and executes the command `/bin/bash` inside of the container.

Therefore _"our"_ command value has to be appended after the image name and before _"our"_ argument values.